docs/extensions/engines/spark/lineage.md

```diff
@@ -220,11 +220,12 @@ Extra works:
 
 Atlas Client configurations (Configure in `atlas-application.properties` or passed in `spark.atlas.` prefix):
 
-|         Name          |  Default Value  |                      Description                      | Since |
-|-----------------------|-----------------|-------------------------------------------------------|-------|
-| atlas.rest.address    | localhost:21000 | The rest endpoint url for the Atlas server            | 1.8.0 |
-| atlas.client.type     | rest            | The client type (currently only supports rest)        | 1.8.0 |
-| atlas.client.username | none            | The client username                                   | 1.8.0 |
-| atlas.client.password | none            | The client password                                   | 1.8.0 |
-| atlas.cluster.name    | primary         | The cluster name to use in qualifiedName of entities. | 1.8.0 |
+|             Name             |  Default Value  | Description                                           | Since |
+|------------------------------|-----------------|-------------------------------------------------------|-------|
+| atlas.rest.address           | localhost:21000 | The rest endpoint url for the Atlas server            | 1.8.0 |
```

Comment:
Whether the protocol should be included as prefix in value?

I found all the default value in Atlas's hook is as the following
`atlas.rest.address=http://localhost:31000`

(https://github.com/apache/atlas/blob/master/addons/hive-bridge/src/test/resources/atlas-application.properties#L20)